lua/stand.lua

```diff
@@ -44,7 +45,37 @@ function M.setup(options)
 
 		timer:stop()
 		timer:again()
-	end, {})
+	end, { "Stand, dismissing any notifications and restarting the timer." })
+
+	vim.api.nvim_create_user_command("StandEvery", function(opts)
+		options.minute_interval = tonumber(opts.args) * 60000
+		local remaining = timer:get_due_in()
+		timer:stop()
+		timer:start(remaining, options.interval, vim.schedule_wrap(on_timer)) -- Start timer with new time
+	end, { desc = "Set the break timer interval in minutes. Will apply to next timer.", nargs = 1 })
```

Comment:
Nitpick to phrase this in a similar tone as the other commands. Thank you for adding descriptions for all the commands :)

```suggestion
	end, { desc = "Set the stand interval in minutes for future timers.", nargs = 1 })
```